### PR TITLE
Disable beta ESAs - version 2

### PR DESCRIPTION
**Pull request by tevans78**
This is a second version of https://github.com/OpenLiberty/open-liberty/pull/22876
#build

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/build.gradle [hunk 1]
```diff
@@ -270,42 +269,47 @@ task publishFeatureResources {
...
-              descriptionHtmlOutputFile: "${buildDir}/repo/description.html",
-              generateKnowledgeCentreLinks: "true",
-              licenseType: "EPL")
-      ant.zip(destfile: "${buildDir}/repo/${symbolicName}.esa.metadata.zip") {
-        fileset(dir: "${buildDir}/repo", includes: "description.html,assetInfo.properties")
-        zipfileset(dir: buildImage.file("lafiles/featureTerms_html"), fullpath: "lafiles/en.html", includes: "en.html")
+      
+      boolean gaFeature = 'ga'.equals(kind)
+      if(gaFeature){
+        String version = ('beta'.equals(kind) ? bnd.libertyBetaVersion : bnd.libertyServiceVersion)
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/build.gradle]:
```suggestion
        String version = bnd.libertyServiceVersion
```

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
Never going to be merged

**Pull request closed without merging**